pkg/providers/imagefamily/bootstrap/aksbootstrap.go

```diff
@@ -145,32 +145,15 @@ var (
 	// NodeTemplate                : user input that could be per-node (t) - ALL DEFAULTED FOR NOW
 	// Provisioner spec            : selected nodepool-level user input (p)
 
-	// NodeBootstrapVariables carries all variables needed to bootstrap a node
+	// staticNodeBootstrapVars carries all variables needed to bootstrap a node
 	// It is used as input rendering the bootstrap script Go template (customDataTemplate)
 	// baseline, covering unused (-), static (s), and unsupported (n) fields,
 	// as well as defaults, cluster/node level (cd/td/xd)
 	staticNodeBootstrapVars = nbcontractv1.Configuration{
-		CustomCloudConfig: &nbcontractv1.CustomCloudConfig{
-			IsAksCustomCloud:  false,              //n
-			InitFilePath:      "",                 //n
-			RepoDepotEndpoint: "",                 //n
-			TargetEnvironment: "AzurePublicCloud", //n
-		},
-		LinuxAdminUsername: "azureuser", // td
-		KubeBinaryConfig: &nbcontractv1.KubeBinaryConfig{
-			KubeBinaryUrl:        "", // cd
-			CustomKubeBinaryUrl:  "", // -
-			PrivateKubeBinaryUrl: "",
-		},
-		KubeproxyUrl: "", // -
-		ApiserverConfig: &nbcontractv1.ApiServerConfig{
+		ApiServerConfig: &nbcontractv1.ApiServerConfig{
```

Comment:
should we get rid of apiserverconfig here since, it is now initialized in apply options?